### DISCUSSION TITLE
Support for Cue Files

### DISCUSSION THREAD

**Discussion by johnledezma34** [PaulWoitaschek/Voice#1370] [Ideas]
can you add Cue file support’ they help distuinguish chapters
Phone Name:htc 10
Android Version:nougat

**Comment by onnyyonn** [comment]
I have some audiobooks with the chapter information in a cue file. It would be great if I could make them work in Voice. Is there any plan for supporting this feature?

**Comment by mmo-onn** [comment]
This would be a great addition
